docs/releases.md

```diff
@@ -12,6 +12,11 @@ Patch:
 * fix `UploadSmallFile` method in `ingestion` module
 * adding `extractPaths` method to `Segmentation` class ([doc](https://github.com/adobe/aepp/blob/main/docs/segmentation.md#extractpaths))
 * adding `extractAudiences` method in `Segmentation` class ([doc](https://github.com/adobe/aepp/blob/main/docs/segmentation.md#extractaudiences))
+Deprecate:
```

Comment:
I think this should be a new section since 0.3.4 was already released. @pitchmuc any preference for what the next version would be and if there are other baked changes not yet released that would be part of it?

Reply:
this part is not yer clear to me, from what i see current version is 0.3.4-1 (and readme was also patched as part of this small bump) I assumed above changes are insignificant and thus don't qualify as separate version, i have nothing against bumping it to 0.3.5.

Reply:
It depends if you see it as a major change. 
As you deprecate some methods, I would do a clear cut to a 0.3.5, usually adding few extra methods is a patch (0.3.4-2) but as just said, it seems more important to do a clear cut on this one and release a 0.3.5.